perses/app/experiments.py

```diff
@@ -878,10 +880,10 @@ def _solvate(self, topology, positions, model = 'tip3p', vacuum = False):
         modeller = app.Modeller(topology, positions)
         hs = [atom for atom in modeller.topology.atoms() if atom.element.symbol in ['H'] and atom.residue.name not in ['MOL','OLD','NEW']]
         modeller.delete(hs)
```

Comment:
i think that stripping and readding hydrogens is problematic.  will make a note to change in `Experiments`

Reply:
Correct! I think @hannahbrucemacdonald has already addressed this in https://github.com/choderalab/perses/pull/616 via https://github.com/choderalab/perses/pull/616/commits/f4cf4355e24290dbd27f6d65f714107f60a7f258 so you can replicate that here.